cfme/containers/project.py

```diff
@@ -22,12 +22,17 @@ class ProjectDetailsView(ContainerObjectDetailsBaseView):
     pass
 
 
+class ProjectDashboardView(ContainerObjectDetailsBaseView):
+    pass
+
+
 @attr.s
 class Project(BaseEntity, Taggable, Labelable):
 
     PLURAL = 'Projects'
     all_view = ProjectAllView
     details_view = ProjectDetailsView
+    dashboard_view = ProjectDashboardView
```

Comment:
I would recommend _not_ defining this here, as it can be easily queried from the navigation destination class's `VIEW` attribute.

There's a method in the Navigate object:
https://github.com/RedHatQE/navmazing/blob/master/navmazing/__init__.py#L77

And examples in the codebase of the query:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/provider.py#L185

Reply:
Agree, remove the dashboard_view definition.